### PR TITLE
Terminal

### PR DESCRIPTION
**Pull request by cees91**

### CODE REVIEW

File: Hotel/src/hotel/BedType.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package hotel;
+
+public enum BedType {
+    Single,
+    DoubleBed,
+    TwoSingle,
+    TwoDoubleOneSingleOneBaby,
+    TwoDoubleFourSIngleTwoBaby
```
Comment (ala-roshdieh) [inline Hotel/src/hotel/BedType.java:8]: Single should have lowercase i

File: Hotel/src/hotel/Hotel.java [hunk 2]
```diff
@@ -1,4 +1,73 @@
...
+        return csvRooms;
+    };
+
+    public void stringify() {
+        for(Rooms room: this.rooms)
+        {
+            System.out.println("room: " + room.roomNumber +", type: "+ room.bedType);
+        }
+    }
+//    public Array showRoomTypes(){
```
Comment (ala-roshdieh) [inline Hotel/src/hotel/Hotel.java:57]: If commented code isn't planned to be used, should be deleted to keep repo tidy :)

**Review by ala-roshdieh — APPROVED** [review]
Aside from a small typo and the commented code, everything looks awesome.

**Review by JorisdeJong-cap — COMMENTED** [review]
There are a lot of project and output files in this commit. Project files are personal preferences that can be different per developer and output files get regenerated everytime you hit build, so keep those out to keep the git repo tidy.

**Pull request merged**